docs/management_api.yml

```diff
@@ -28,83 +28,80 @@ tags:
   - name: ManagementAPI
 
 paths:
-  /configuration:
+  /config/devices/{deviceId}:
     get:
-      operationId: List Parameters
+      operationId: Get key-value pair store
       tags:
         - ManagementAPI
-      summary: Get existing parameters
-      # parameters: {}
+      summary: Query the configuration store; retrieve all key-value pairs
+      parameters:
+        - in: path
+          name: deviceId
+          schema:
+            type: string
+          required: true
+          description: ID of the device to query.
       responses:
         200:
           description: OK
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  $ref: '#/components/schemas/Parameter'
+                type: object
+                properties:
+                  id:
+                    type: string
+                  expected:
```

Comment:
About naming: to be honest, I don't like the name `expected`. I would prefer `required`, or maybe `config` or `configuration`. The `expected` term is a bit out of context here.

Reply:
expected and actual are the names that I had agreed upon with Manuel before and at that time I do not think this code was merged. I can change that no problem.

Reply:
ok, I lied a bit: I have not changed that. do you think we can keep expected/actual? I agreed on it with Manuel.